### PR TITLE
Fixes #6614: cf-serverd.cf is broken because of missing promise type

### PR DESCRIPTION
**Pull request by ncharles**

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
"access:" promise type should be moved outside the "if" and we shoudl laways define access to /var/rudder/cfengine-community/bin/cf-agent

**Comment by ncharles** [comment]
yeah i'm on it
the bug is bigger than that (massive slowdon in promise generation)

**Comment by VinceMacBuche** [comment]
:( :(

File: techniques/system/common/1.0/cf-served.st [hunk 1]
```diff
@@ -64,31 +63,32 @@ bundle server access_rules
...
         admit   => { host2ip("&POLICYSERVER&"), escape("&POLICYSERVER&")  };
-&endif&
-    any::
+  &endif&
+
+    policy_server_set::
       # Allow server to remotely run the agent
       "/var/rudder/cfengine-community/bin/cf-agent"
-        admit   => { host2ip("${server_info.cfserved}"), escape("${server_info.cfserved}") };
+        admit   => { host2ip("${def.policy_server}"), escape("${def.policy_server}") };
```
Comment (ncharles) [inline techniques/system/common/1.0/cf-served.st]: there is no reason to use a different variable than in l123

File: techniques/system/common/1.0/cf-served.st [hunk 2]
```diff
@@ -118,7 +118,7 @@ bundle common def
       &AUTHORIZED_NETWORKS:{net|"&net&",}&
     };
 &endif&
-    !policy_server::
+    !policy_server.policy_server_set::
```
Comment (peckpeck) [inline techniques/system/common/1.0/cf-served.st]: The  policy_server_set class is not defined in branch 2.10

File: techniques/system/common/1.0/cf-served.st (hunk 1 above)
Comment (peckpeck) [inline techniques/system/common/1.0/cf-served.st]: The initial promise cf-served.cf and all other files use server_info.cfserved shouldn't we correct cf-serverd.st  instead ?

Comment (ncharles) [inline techniques/system/common/1.0/cf-served.st]: Oh, this part makes no sense in 2.10, as we have "${sys.workdir}"  maproot => { host2ip("&POLICYSERVER&"), escape("&POLICYSERVER&") },

File: techniques/system/common/1.0/cf-served.st (hunk 2 above)
Comment (ncharles) [inline techniques/system/common/1.0/cf-served.st]: indeed ...

**Comment by ncharles** [comment]
this PR was replaced by https://github.com/Normation/rudder-techniques/pull/673

**Pull request closed without merging**